lua/starfall/libs_sh/players.lua

```diff
@@ -236,6 +236,13 @@ function player_methods:getDuckSpeed()
 	return getply(self):GetDuckSpeed()
 end
 
+--- Returns a list of valid playermodels
+-- @shared
+-- @return table List of valid playermodels
+function player_methods.AllValidModels()
+	return table.Add(nil, player_manager.AllValidModels())
+end
+
```

Comment:
You forgot to remove the old function from players lib.

Reply:
fixed